charts/dbildungscloud-iam-keycloak/values.yaml

```diff
@@ -1,5 +1,4 @@
 keycloakHostname: "keycloak.dev.spsh.dbildungsplattform.de"
-keycloakPublic: false
 secrets:
   name: dbildungs-iam-secrets
```

Comment:
I think it is better if we do not use the dbildungs-iam-secrets for the keycloak service. We can append the desired secrets into the keycloak secret where KEYCLOAK_ADMIN_PASSWORD and KEYCLOAK_ADMIN is also defined.